public/cli/lib/commands/ffmpeg.ts

```diff
@@ -8,6 +8,25 @@ import { paths } from '../path-constants';
 
 export async function combineFrames(settings: FfmpegSettings): Promise<void> {
   const executeAudioPath = await combineAudioIfNecessary(settings.audioFiles);
+  
+  // check if output file already exists:
+  var outputName = settings.outputName;
```

Comment:
Don't use `var` anymore (ESLint will complain if you have the extension enabled in VSCode). Use `let` for variables that get reassigned and `const` everywhere else. Here and below.